src/basic/cgroup-util.c

```diff
@@ -114,9 +118,7 @@ int cg_read_pid(FILE *f, pid_t *ret) {
                 return errno_or_else(EIO);
         }
 
-        if (ul <= 0)
-                return -EIO;
-        if (ul > PID_T_MAX)
+        if (ul < 0 || ul > PID_T_MAX)
```

Comment:
Isn't the <0 check superfluous, given it's an unsigned long?

Reply:
It is. I should have compiled :)